.gitignore

```diff
@@ -1,6 +1,5 @@
 /target
 /pkg
 /private
-.vscode
```

Comment:
I expect you have a `.vscode` as well, so 🤷 

I needed to set the `rust-analyzer` settings for it to pick up the Cargo.toml that isn't a member of the root workspace

Reply:
Ah I see. Would adding it to the [top level Cargo workspace](https://github.com/kaleidawave/ezno/blob/c834e102e59a0eed7e5b71ee9d24d3226e2dd796/Cargo.toml#L2) 1) fix that and 2) have any downsides?

Reply:
from what I understand it's intentionally not part of the base Cargo workspace - that's why the Cargo.toml has workspace.members=["."] in it. 

I wish I knew precisely why it was a problem though

Reply:
@VTCAKAVSMoACE do you have a good explanation for why the fuzz workspace shouldn't be a member of the root workspace?

Reply:
In many situations, you don't want the fuzz target dir to be shared because the fuzzer is generally built with [sanitizers](https://doc.rust-lang.org/beta/unstable-book/compiler-flags/sanitizer.html).

Reply:
Ah so if it got added the main workspace the assets would end up in the top level `target` directory and that would break whatever magic `cargo fuzz` needs to do to built assets.

Reply:
Yup, precisely. At best, it would just make the build times worse. At worse, you might have malformed builds :slightly_smiling_face: